### PR TITLE
Presentation 

### PR DESCRIPTION
**Pull request by ghostaccount737**

### CODE REVIEW

File: docs/gsoc_project_progress.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Setting Up Continuos Integration
+- Coming up With System Architecture
+- Data Ingestion Service
+- Data Filtering Service
+- Containerizing Ingestion and Filtering Service
+- Semi-Parser(Generates Measures)
+
+## Description
+
+Currently,VizRec takes selected json files and ingests it into the mongo database . From the database, the filtering service generates the schema files that can be used to have different chart types like bar,pie,line,table..etc.However it can generate a fewer parameters currently because of the limited number of aggregates and measures enabled in the schema compiler of cube.dev and are looking to automate it in the future
```
Comment (lastlegion) [inline docs/gsoc_project_progress.md]: Run through grammar check, lots of errors.

Comment (ghostaccount737) [reply docs/gsoc_project_progress.md]: Updated

File: docs/gsoc_project_progress.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Setting Up Continuos Integration
+- Coming up With System Architecture
+- Data Ingestion Service
+- Data Filtering Service
+- Containerizing Ingestion and Filtering Service
+- Semi-Parser(Generates Measures)
+
+## Description
+
+Currently, the ingestion service of VizRec accepts well-formatted JSON files as input and ingests it into the mongo database. The filtering service of VizRec generates the schema files from the database. Schema Files comprise of measures, dimensions, filter, joins, and pre-aggregations. One can visualize different chart types such as bar, pie, line, table..etc from the schema files
```
Comment (lastlegion) [inline docs/gsoc_project_progress.md]: _table etc._

File: README.md [hunk 3]
```diff
@@ -1,2 +1,42 @@
-# VizRec
-Deep learning based visualization recommendation.
+
+<h1 align="center">Deep Learning Guided Visual Exploration of Data</h1>
```
Comment (lastlegion) [inline README.md]: use either markdown or HTML, don't mix

File: docs/gsoc_project_progress.md (hunk 2 above)
Comment (lastlegion) [reply docs/gsoc_project_progress.md]: @PVinay737 ^

Comment (ghostaccount737) [reply docs/gsoc_project_progress.md]: Removed the `from the schema files`

Comment (lastlegion) [reply docs/gsoc_project_progress.md]: table..etc => table etc.

Comment (ghostaccount737) [reply docs/gsoc_project_progress.md]: Updated

**Review by lastlegion — APPROVED** [review]

**Pull request merged**